content/en/docs/setup/production-environment/windows/user-guide-windows-nodes.md

```diff
@@ -102,35 +77,7 @@ Once you have a Linux-based Kubernetes control-plane ("Master") node you are rea
     {{< note >}}The VNI must be set to 4096 and port 4789 for Flannel on Linux to interoperate with Flannel on Windows. Support for other VNIs is coming soon. See the [VXLAN documentation](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan)
```

Comment:
I'd also change to:
```suggestion
    {{< note >}}The VNI must be set to 4096 and port 4789 for Flannel on Linux to interoperate with Flannel on Windows. See the [VXLAN documentation](https://github.com/coreos/flannel/blob/master/Documentation/backends.md#vxlan).
```
based on https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future